.gitignore

```diff
@@ -17,7 +17,6 @@ build/
 expected/
 notes/
 baserom/
-baseroms/*/segments/
```

Comment:
we could possibly make a "legacy" section of the gitignore and keep both of these baserom paths there? 

the idea behind a legacy section is that it would eventually be deleted, but it gives people time to stop working with old branches that use the old folder structure